qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -34,7 +34,7 @@
                                urlutils, message)
 
 
-UndoEntry = collections.namedtuple('UndoEntry', ['url', 'history', 'index'])
+UndoEntry = collections.namedtuple('UndoEntry', ['url', 'history', 'index', 'pinned'])
```

Comment:
Seems fine to me. I guess we could save the whole `TabData` object instead, but since we don't need anything else from it, I think this is the best way.